.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+max-line-length = 120
+max-complexity = 18
+exclude = 
+  notebooks/* 
+  */.ipynb_checkpoints/*
```

Comment:
This will only ignore any `.ipynb_checkpoints` in your root directory. If you want to exclude *all* `.ipynb_checkpoints` you'll want to put `**/.ipynb_checkpoints/*`